model/ftn/w3odatmd.ftn

```diff
@@ -758,7 +760,8 @@
       IDOUT( 5, 8)  = 'Whitecap mean thick.'
       IDOUT( 5, 9)  = 'Mean breaking height'
       IDOUT( 5,10)  = 'Dominant break prob '
-!      IDOUT( 5,11)  = 'Breaker passage rate'
+      IDOUT( 5,11)  = 'Wind sea period' ! C.Bunney - reinstated this as is used in ww3_ounf
+                                        ! Is it suposed to be defunct? It is not in ww3_outf...
```

Comment:
This parameter is not currently listed in the manual? However, I think @ukmo-juan-castillo has mentioned it previously as a coupled model requirement? Suggest this isn't addressed here, but worth resolving in issue #206

Reply:
> This parameter is not currently listed in the manual? However, I think @ukmo-juan-castillo has mentioned it previously as a coupled model requirement? Suggest this isn't addressed here, but worth resolving in issue #206

This parameter is not required for coupling at the moment, and I do not think it will.

Reply:
wind sea period was introduced by Fabrice Ardhuin. If it's not explained in the manual, it would be good to create an issue to remember to do it

Reply:
Also - it is an output field in ww3_ounf, but NOT in ww3_outf.

Confusingly, though, in field name matching subroutine in w3iogomd, the internal field name for IFI=5, IFJ=11 is `FWS`, even though it writes out field `TWS` (line 917)

`TWS` is actually IFI=4, IFJ=16 (line 872) - bit this seems wrong as later this is associated with partitioned wind sea fraction (line 2987).

I think those two parameters have become switched at some point.

I agree that an issue needs raising to sort out this! I don't want to change that stuff here as it will affect my regtests and it feels like a separate bug fix.

Reply:
Issue raised for the TWS/FWS issue here: https://github.com/NOAA-EMC/WW3/issues/267